app/model/sentiment_analysis.py

```diff
@@ -180,78 +193,129 @@ def get_data_from_transcribe_table(self, server_name, database_name, client_id,a
+                # result.close()
+            finally:
+                self.logger.log_entry_into_sql_table(server_name, database_name, client_id, True)
+                session.close()
+        else:
+            result = {'status': INTERNAL_SERVER_ERROR, "message": "Unable to connect to the database"}
+            return result,INTERNAL_SERVER_ERROR
 
     def get_sentiment_data_from_table(self, server_name, database_name, client_id,audio_file):
         self.logger.log_entry_into_sql_table(server_name, database_name, client_id, False)
         connection_string = self.global_utility.get_connection_string(server_name, database_name, client_id)
-        session = self.global_utility.get_database_session(connection_string)
+        if len(connection_string) > 0:
+            session = self.global_utility.get_database_session(connection_string)
+            check_audio_file_exits = session.query(SentimentAnalysis).filter(
+                SentimentAnalysis.AudioFileName == audio_file).all()
 
-        try:
-            sentiment_dic={}
-            data = session.query(SentimentAnalysis).filter_by(AudioFileName=audio_file).all()
-            sentiment_dic.update({"Id":data[0].Id,"ClientId":data[0].ClientId,
-                                  "AnalysisDateTime":data[0].AnalysisDateTime,"AudioFileName":data[0].AudioFileName,
-                                  "Created":data[0].Created,"SummaryReport":data[0].Summary,"Topics":data[0].Topics,
-                                  "FoulLanguage":data[0].FoulLanguage,"ActionItems":data[0].ActionItems,
-                                  "Owners":data[0].Owners,"SentimentScore":data[0].SentimentScore,
-                                  "SentimentStatus":data[0].SentimentStatus,
-                                  "Modified":data[0].Modified,"Sentiment":data[0].Sentiment})
-            result = {"sentimentdata": sentiment_dic,"status": 200}
-            return result
-        except Exception as e:
-            # self.logger.error(f": Error {e}",e)
-            print(e)
-        finally:
-            self.logger.log_entry_into_sql_table(server_name, database_name, client_id, True)
-            session.close()
+            try:
+                if len(check_audio_file_exits) > 0:
+                    sentiment_dic={}
+                    data = session.query(SentimentAnalysis).filter_by(AudioFileName=audio_file).all()
+                    sentiment_dic.update({"Id":data[0].Id,"ClientId":data[0].ClientId,
+                                          "AnalysisDateTime":data[0].AnalysisDateTime,"AudioFileName":data[0].AudioFileName,
+                                          "Created":data[0].Created,"SummaryReport":data[0].Summary,"Topics":data[0].Topics,
+                                          "FoulLanguage":data[0].FoulLanguage,"ActionItems":data[0].ActionItems,
+                                          "Owners":data[0].Owners,"SentimentScore":data[0].SentimentScore,
```

Comment:
suppose some value is not return by API then this code will be break. Apply the undefine and null check here